workflows/transcriptomics/goseq/goseq-go-kegg-enrichment-analsis-tests.yml

```diff
@@ -0,0 +1,101 @@
+- doc: Test outline for goseq-go-kegg-enrichment-analsis
+  job:
+    DE result:
+      class: File
+      path: test-data/de-result.tabular
+      filetype: tabular
+    gene length:
+      class: File
+      path: test-data/gene-length.tabular
+      filetype: tabular
+    KEGG pathways:
+      class: File
+      path: test-data/kegg-pathways.tabular
+      filetype: tabular
+  outputs:
+    goseq-table-cc:
```

Comment:
These are the primary outputs, can you adjust them (here and in the workflow) to be human readable, i.e. no `-`, `_` and abbreviations that aren't well known ? I don't know what CC is here